docs/deploy-to-netlify.md

```diff
@@ -8,4 +8,14 @@ To deploy your Gridsome site to Netlify, go to the create a new site page, selec
 
 Add these build settings:
 - **Build Command:** `gridsome build`
-- **Publish directory:** `dist`
\ No newline at end of file
+- **Publish directory:** `dist`
+
+Alternative deploy using toml file:
+  ```
+  [build]
+    publish = "dist"
+    command = "gridsome build"
+  ```
+ 
+ Create a file in the root of your project called `netlify.toml` this file should have the above configuation.
```

Comment:
Thoughts on adding a "For more configuration via `.toml` check here" with a link to https://www.netlify.com/docs/netlify-toml-reference/ ?

Reply:
I've made this change